src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -149,6 +162,36 @@ impl RuntimeHandlerManagerInner {
 
         let config = load_config(spec, options).context("load config")?;
 
+        // Retrieve the log-levels set in configuration file, modify the FILTER_RULE accordingly
+        let mut agent_level = "".to_string();
+        for v in config.agent.values() {
+            agent_level = v.log_level.clone();
+        }
+        let mut hypervisor_level = "".to_string();
+        for v in config.hypervisor.values() {
+            hypervisor_level = v.debug_info.log_level.clone();
+        }
+        let runtime_level = config.runtime.log_level.clone();
+
+        // Update FILTER_RULE to apply changes
+        FILTER_RULE.rcu(|inner| {
+            let mut updated_inner = HashMap::new();
+            updated_inner.clone_from(inner);
+            updated_inner.insert(
+                "runtimes".to_string(),
+                convert_string_to_slog_level(runtime_level.clone()),
```

Comment:
I think you don't need to clone `runtime_level` again as it won't be used anymore in this function